models/payout.go

```diff
@@ -56,6 +56,19 @@ func AllocatePayouts() error {
 			return err
 		}
 
+		for _, l := range ledgerEntries {
+			if l.authorId == user.ID {
+				// deposit ledgers
+				return createLedgerEntry(user.ID, l.contributorId, user.Cash, l.ledgerType, l.description)
+			}
+		}
+
+		//withdrawal ledger entry
+		err = createLedgerEntry(user.ID, -1, user.Cash, "withdrawal", "withdrawal from non.io to Stripe")
```

Comment:
This looks correct, though just for consistency can you have this be right after the db call (the one on line 49)? Also is it possible to make the two calls (updating the cash amount and creating a ledger entry) a transaction, such that if any single one fails, they both do?

Reply:
We would need to build a bulk insert statement(basically a string builder) if we want to introduce these db calls as a transaction. Do you want me to do that? It is obviously do-able.